MARKDOWN-GUIDE.md

```diff
@@ -0,0 +1,124 @@
+# Markdown Syntax
+This guide gives a quick overview over all markdown options that are available within Secvisogram.
+
+## Headings
+To create a heading, up to six number signs (#) can be added before the heading text. The number of  number signs (#) used determines the hierarchy level and the font size of the heading.
+```
+# A first-level heading
+## A second-level heading
+### A third-level heading
+```
+
+## Format Text
+| Style                         | Syntax               | Example                                  | Output                                 |
+|-------------------------------|----------------------|------------------------------------------|----------------------------------------|
+| Bold                          | `** **` or `__ __`   | `**Bold Text**`                          | **Bold Text**                          |
+| Italic                        | `* *` or `_ _`       | `_Italic Text_`                          | _Italic Text_                          |
+| strikethrough                 | `~~ ~~`              | `~~This was mistaken Text~~`             | ~~This was mistaken Text~~             |
+| Nested Bold and Italic        | `** **` and `_ _`    | `**This text is _extremely_ important**` | **This text is _extremely_ important** |
+| All Bold and Italic           | `*** ***`            | `***All this text is important***`       | ***All this text is important***       |
+
+## Quoting Text
+Text can be quoted with the following symbol: `>`.
+
+**Example:**
+```
+not quoted text
+> quoted text
+```
+**Output:**
```

Comment:
Please review these structures as it looks weird (see https://github.com/BIZFactoryGmbH/secvisogram/blob/feat/markdown-preview/MARKDOWN-GUIDE.md#links as example)

BTW: Italic might be a better choice.